src/test/java/org/broadinstitute/hellbender/tools/walkers/genotyper/afcalc/AlleleFrequencyCalculatorUnitTest.java

```diff
@@ -170,6 +170,25 @@ public void testManyRefSamplesDontKillGoodVariant() {
         }
     }
 
+    @Test
```

Comment:
I don't think you meant to include this test.

Reply:
I wondered why I didn't have to stash the test when I checked out my other branch.

I removed the semicolons.  Maybe that was what kept the old template from actually being rendered in markdown.